packages/emails/templates/attendee-was-requested-to-reschedule-email.ts

```diff
@@ -42,6 +43,8 @@ export default class AttendeeWasRequestedToRescheduleEmail extends OrganizerSche
   // @OVERRIDE
   protected getiCalEventAsString(): string | undefined {
     const icsEvent = createEvent({
+      uid: this.calEvent.iCalUID || this.calEvent.uid!,
```

Comment:
`nitpick`: I'm tending to start telling people to use nullish coalescing, eg:
```suggestion
      uid: this.calEvent.iCalUID ?? this.calEvent.uid!,
```

Reply:
Also, do we have no guarantee of `this.calEvent.uid`?

Reply:
It's interesting. I've seen this in our codebase before. Looking at the schema, `uid` is required when saving bookings. Although the `CalendarEvent` type we often use in the code base sets `uid` as possibly undefined.